bundles/org.openhab.binding.tradfri/src/main/resources/OH-INF/i18n/tradfri.properties

```diff
@@ -1,51 +1,81 @@
-
-# thing types
-
-thing-type.tradfri.0010.label = On/Off Plug
-thing-type.tradfri.0010.description = A plug that can be switched on and off.
-thing-type.tradfri.0100.label = Dimmable Light
-thing-type.tradfri.0100.description = A light that has continuous brightness control.
-thing-type.tradfri.0107.label = Occupancy Sensor
-thing-type.tradfri.0107.description = This represents the motion sensor capable of reporting the battery level.
-thing-type.tradfri.0202.label = Window Covering Device
-thing-type.tradfri.0202.description = This represents a blind or curtain that can be moved up and down. Also reports current battery level.
-thing-type.tradfri.0203.label = Window Covering Controller
-thing-type.tradfri.0203.description = This represents the wireless open/close remote capable of reporting the battery level.
-thing-type.tradfri.0210.label = Color Light
-thing-type.tradfri.0210.description = A dimmable light that supports full colors and color temperature settings.
-thing-type.tradfri.0220.label = Color Temperature Light
-thing-type.tradfri.0220.description = A dimmable light that supports different color temperature settings.
-thing-type.tradfri.0820.label = Non-Colour Controller
-thing-type.tradfri.0820.description = This represents the wireless dimmer sensor capable of reporting the battery level.
-thing-type.tradfri.0830.label = Non-Colour Scene Controller
-thing-type.tradfri.0830.description = This represents the remote control capable of reporting the battery level.
-thing-type.tradfri.gateway.label = TRÅDFRI Gateway
-thing-type.tradfri.gateway.description = IKEA TRÅDFRI IP Gateway
-
-# thing types config
-
-bridge-type.config.tradfri.gateway.code.label = Security Code
-bridge-type.config.tradfri.gateway.code.description = Security code printed on the label underneath the gateway.
-bridge-type.config.tradfri.gateway.host.label = Host
-bridge-type.config.tradfri.gateway.host.description = Hostname or IP address of the IKEA TRÅDFRI gateway.
-bridge-type.config.tradfri.gateway.identity.label = Identity
-bridge-type.config.tradfri.gateway.identity.description = Unique identity used for communication with the gateway.
-bridge-type.config.tradfri.gateway.port.label = Port
-bridge-type.config.tradfri.gateway.port.description = Port for accessing the gateway.
-bridge-type.config.tradfri.gateway.preSharedKey.label = Pre-Shared Security Key
-bridge-type.config.tradfri.gateway.preSharedKey.description = Security key obtained during first initialization of the gateway.
-thing-type.config.tradfri.device.id.label = ID
-thing-type.config.tradfri.device.id.description = The identifier of the device on the gateway.
-
-# channel types
-
-channel-type.tradfri.position.label = Position
-channel-type.tradfri.position.description = Control the position of the blind or curtain in percent from 0 (open) to 100 (closed).
-
-# discovery result
-
-discovery.gateway.label = TRÅDFRI Gateway
+# add-on
+
+addon.tradfri.name = TRÅDFRI Binding

```

Comment:
It seems something went wrong with the formatting or character set of this file. In GitHub it looks like all line endings have changed.

Reply:
Strangely enough this happened when running `mvn i18n:generate-default-translations` as per documentation.
My assumption is this is because I am on a Windows environment, thought I guess the `mvn i18n:generate-default-translations` should ensure a unified line ending is used irrespective of the development environment.
Not sure if this is a big but I will revert and add translations by hand instead of using this target.

Reply:
Resolved by reverting to original file and adding translations without line ending issues